services/wfm/src/main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/SpeakerRequestBolt.java

```diff
@@ -86,7 +86,7 @@ private void processNotFoundError(CommandMessage commandMessage, SwitchId switch
         ErrorMessage errorMessage = new ErrorMessage(errorData, System.currentTimeMillis(),
                 commandMessage.getCorrelationId(), null);
         Values values = new Values(commandMessage.getCorrelationId(), errorMessage);
-        if (commandMessage.getData() instanceof ValidateRulesRequest) {
+        if (commandMessage.getData() instanceof DumpRulesForNbworkerRequest) {
```

Comment:
`SpeakerRequestBolt` do not declare Stream.NORTHBOUND_REPLY (starting from this change). So will the attempt to write into this stream lead to an error on storm level?